added several missing diodes

jkriege2 opened the pull request:
- basics:
  - only new (required) footprint: DO-201AE: https://github.com/KiCad/kicad-footprints/pull/404
  - all symbols based on existing standard symbols from device.lib (therefore no screenshots)
  - as done with all other zener diodes in the lib, the zener voltage rating (and TVS voltage ratings for that matter) were wildcarded out (xx/xxx)
  - remaining KLC-violations due to not-yet-merged footprint-PR (DO-201AE) or due to the script not recognizing multiple single diodes in a package as "small devices", which they are, so please ignore the latter

- devices:
  - 1.5KExxA: https://www.vishay.com/docs/88301/15ke.pdf
  - 1.5KExxCA: https://www.vishay.com/docs/88301/15ke.pdf
  - 1N47xxA: https://www.vishay.com/docs/85816/1n4728a.pdf
  - 1N53xxB: https://diotec.com/tl_files/diotec/files/pdf/datasheets/1n5345b.pdf
  - 1N5711: https://www.microsemi.com/document-portal/doc_download/8865-lds-0040-datasheet
  - 1N5711UR: https://www.microsemi.com/document-portal/doc_download/131890-lds-0040-1-datasheet
  - 1N5712: https://www.microsemi.com/document-portal/doc_download/8865-lds-0040-datasheet
  - 1N5712UR: https://www.microsemi.com/document-portal/doc_download/131890-lds-0040-1-datasheet
  - 1N62xxA: https://www.vishay.com/docs/88301/15ke.pdf
  - 1N62xxCA: https://www.vishay.com/docs/88301/15ke.pdf
  - 1N630xA: https://www.vishay.com/docs/88301/15ke.pdf
  - 1N630xCA: https://www.vishay.com/docs/88301/15ke.pdf
  - 1N6857: https://www.microsemi.com/document-portal/doc_download/8865-lds-0040-datasheet
  - 1N6857UR: https://www.microsemi.com/document-portal/doc_download/131890-lds-0040-1-datasheet
  - 1N6858: https://www.microsemi.com/document-portal/doc_download/8865-lds-0040-datasheet
  - 1N6858UR: https://www.microsemi.com/document-portal/doc_download/131890-lds-0040-1-datasheet
  - 2BZX84Cxx: https://diotec.com/tl_files/diotec/files/pdf/datasheets/bzx84c2v4.pdf
  - 5KPxxA: https://diotec.com/tl_files/diotec/files/pdf/datasheets/5kp65.pdf
  - 5KPxxCA: https://diotec.com/tl_files/diotec/files/pdf/datasheets/5kp65.pdf
  - BZT52Bxx: https://diotec.com/tl_files/diotec/files/pdf/datasheets/bzt52b2v4.pdf
  - BZX84Cxx: https://diotec.com/tl_files/diotec/files/pdf/datasheets/bzx84c2v4.pdf
  - DSB2810: https://www.microsemi.com/document-portal/doc_download/8865-lds-0040-datasheet
  - DSB5712: https://www.microsemi.com/document-portal/doc_download/8865-lds-0040-datasheet
  - MM3Zxx: https://diotec.com/tl_files/diotec/files/pdf/datasheets/mm3z2v4.pdf
  - MM5Zxx: https://diotec.com/tl_files/diotec/files/pdf/datasheets/mm5z2v4.pdf
  - MMSD914: https://www.onsemi.com/pub/Collateral/MMSD914T1-D.PDF
  - PMEG6010CEH: https://assets.nexperia.com/documents/data-sheet/PMEG6010CEH_PMEG6010CEJ.pdf
  - PMEG6010CEJ: https://assets.nexperia.com/documents/data-sheet/PMEG6010CEH_PMEG6010CEJ.pdf
  - SMZxxx: https://diotec.com/tl_files/diotec/files/pdf/datasheets/smz1.pdf
  - Z1SMAxxx: https://diotec.com/tl_files/diotec/files/pdf/datasheets/z1sma1.pdf
  - Z2SMBxxx: https://diotec.com/tl_files/diotec/files/pdf/datasheets/z2smb1.pdf
  - Z3SMCxxx: https://diotec.com/tl_files/diotec/files/pdf/datasheets/z3smc1.pdf
  - MMxx: https://diotec.com/tl_files/diotec/files/pdf/datasheets/zmm1.pdf
  - ZYxxx: https://diotec.com/tl_files/diotec/files/pdf/datasheets/zy1


------------
Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the symbol(s) you are contributing
- [x] An example screenshot image is very helpful
- [x] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
- [x] If there are matching footprint PRs, provide link(s) as appropriate
- [x] Check the output of the Travis automated check scripts - fix any errors as required

evanshultz commented:
Great work as always!
- 5KPxxA and 5KPxxCA have different packages in the DCM, but the should be the same from the datasheet. And the symbols have the same default footprint.
- And how did you choose this footprint for this diode family? The datasheet doesn't mention a P-600 package anywhere I can see.
- Z1SMAxxx has two different names for the same package, but the style is the same as if we had `DIP-8/SOIC-8`, which are different packages. The footprint is called `D_SMA` with a description of `Diode SMA (DO-214AC)`, so perhaps change the DCM to call this footprint `SMA (DO-214AC)`?

jkriege2 commented:
Hi!

thanks for the review:
- ad 1): I fixed that
- ad 2): It actually does on the first page (says ~P600):
![2018-03-04 17_56_31-5kp65 pdf geschutzt - adobe acrobat reader dc](https://user-images.githubusercontent.com/11967523/36948076-95ea0532-1fd5-11e8-9cd2-4fc505769cee.png)
- ad 3): fixed

Thanks for the review,
JAN

evanshultz commented:
Works for me. Thanks!
